modules/ding_base/ding_base.module

```diff
@@ -138,7 +138,7 @@ function ding_base_date_formats() {
     ),
     array(
       'type' => 'ding_event_lists_date',
-      'format' => 'l j. F Y',
+      'format' => 'l d F Y',
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-1110

Reply:
I doubt that this is feasible. Please locate where `ding_event_lists_date` is used.

Reply:
https://github.com/easySuite/ding2/pull/664/files#diff-72c29a99dbfdb4c9258f1b00b699ecbaf09fa1156a207b741fbf2f965b20ec4fR250

Reply:
I mean visually in a normal easyOPAC installation. Links or screenshots.

Reply:
Please define a bullet-list of where `ding_event_lists_date` is used in default distribution.

Reply:
"Sunday 5. January 1999" -> "Sunday 05 January 1999"

Reply:
Only place I see it's referenced is the events calendar, the calendar header.

Reply:
**This change should not be ported.**